entrypoint.sh

```diff
@@ -53,12 +54,19 @@ then
         $SCORECARD_BIN --local . --format "$SCORECARD_RESULTS_FORMAT" --show-details --policy "$SCORECARD_POLICY_FILE" > "$SCORECARD_RESULTS_FILE"
     fi
 else
-    # For push events, we run on the repo.
+    # For other events, we run on the repo.
```

Comment:
For simplicity, why not just run a full run? It'll avoid the additional complexity of https://github.com/ossf/scorecard/pull/1379

Reply:
well, we may try to enable branch protection on PR, and the change in https://github.com/ossf/scorecard/pull/1379 is useful for that too. I think the code change is not too complex, but if you disagree we can potentially revert.